dnf5/config/etc/dnf/dnf.conf

```diff
@@ -0,0 +1,3 @@
+# see `man dnf.conf` for defaults and possible options
+
+[main]
```

Comment:
This is different compared to the default `dnf.conf` in the `dnf` component. Doesn't it mean, that when deploying a fresh system having the dnf5 already as the default, then the users would have different configuration than those upgrading from `dnf`?

Reply:
Yes, that was one of the requirements in https://github.com/rpm-software-management/dnf5/issues/417. My understanding is that we used to have distributions set their default config values via providing their own dnf.conf, but now in DNF 5 there is a different mechanism for distribution overrides so we no longer need to prepopulate dnf.conf.

In general, I think it is best to leave default values out of configuration files, or keep them commented out. That way, if we want to change the defaults, we can do so in e.g. `config_main.cpp`, and a user will only have an option set in dnf.conf if they have set it themselves.